frontend/index.html

```diff
@@ -2,9 +2,8 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React + TS</title>
+    <title>QuackStack NMP Calculator</title>
```

Comment:
```suggestion
    <title>Nutrient Management Calculator</title>
```
Sorry :) Business facing parts should maintain a closer replica to the original

Reply:
Np I'll edit that. Thank you.